.htmltest.yml

```diff
@@ -8,6 +8,7 @@ TestFilesConcurrently: true
 IgnoreDirs:
   - ^blog/(\d+/)?page/\d+
 IgnoreInternalURLs: # list of paths
+  - /docs/contribution-guidelines/
```

Comment:
Ah, if you modify the community list shortcode, then you don't need to ignore the path anymore:

```suggestion
```